src/fontra/client/web-components/range-slider.js

```diff
@@ -242,7 +277,24 @@ export class RangeSlider extends html.UnlitElement {
     this.rangeInput.classList.toggle("is-at-default", this.value == this.defaultValue);
   }
 
+  isDiscrete() {
+    return this.values.length > 0;
+  }
+
   render() {
+    const isDiscrete = this.isDiscrete();
+    let minValue, maxValue, step, value;
+    if (isDiscrete) {
+      minValue = 0;
+      maxValue = this.values.length - 1;
+      step = 1;
+      value = this.valueFormatted;
+    } else {
+      step = this.step;
+      minValue = this.minValue;
+      maxValue = this.maxValue;
+      value = this.getClosestDiscreteValue(this.value);
```

Comment:
Hm, this is happening in the else branch of the `isDiscrete` if. Shouldn't line 291 and 296 be swapped?

Reply:
Yes, it's wrong. I sent a fix.